File: Question1/mario-more/mario.c
#include <cs50.h>
#include <ctype.h>
#include <stdio.h>
bool isInTheRange(int height);
int main(void)
{
    int height = 0;
    do
    {
        height = get_int("Height: ");
    }
    while (!isInTheRange(height));
    for (int i = 0; i < height; i++)
    {
        for (int j = 0; j < (height - 1 - i); j++)
        {
            printf(" ");
        }
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("  ");
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}
bool isInTheRange(int height)
{
    if (height <= 8 && height >= 1)
    {
        return true;
    }
    return false;
}

File: Class6/mario.py
from cs50 import *

def main():
    height = get_height()
    for i in range(height):
        print("#")

def get_height():
    while True:
        # get_int()这个函数必须要接收参数，不能空
        n = get_int("Height: ")
        if n > 0:
            return n

main()
